go.mod

```diff
@@ -25,6 +25,7 @@ replace (
 )
 
 replace (
+	git.apache.org/thrift.git => github.com/apache/thrift v0.0.0-20180902110319-2566ecd5d999
```

Comment:
why do we need this? aren't they both the same and allows for thrift.git to be be moved anywhere?

Reply:
I don't know, I hit the dependency resolve issue when run go mod, and then find this:
https://github.com/jenkins-x/jx/issues/3320
it resolve my issue.